### PR TITLE
mapper+test+get+post

### PR DESCRIPTION
**Pull request by sealDebolina**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-3425

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts [hunk 1]
```diff
@@ -57,21 +93,46 @@ export class ConfirmContactDetailsPostController extends PostController<AnyObjec
...
+              ...req.session.userCase.applicantsFL401.response,
+              ...response,
+            },
+          };
         }
       }
     }
 
     const caseData = toApiFormat(req?.session?.userCase);
+    console.log(caseData);
```
Comment (vivek-sekhar) [inline src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts:135]: Remove console statements

File: yarn-audit-known-issues [hunk 2]
```diff
@@ -0,0 +1 @@
...ble_versions":"<0.5.0","module_name":"xml2js","severity":"high","github_advisory_id":"GHSA-776f-qx25-q3cc","cves":["CVE-2023-0842"],"access":"public","patched_versions":">=0.5.0","cvss":{"score":7.3,"vectorString":"CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L"},"updated":"2023-04-10T19:01:30.000Z","recommendation":"Upgrade to version 0.5.0 or later","cwe":["CWE-1321"],"found_by":null,"deleted":null,"id":1091653,"references":"- https://nvd.nist.gov/vuln/detail/CVE-2023-0842\n- https://fluidattacks.com/advisories/myers/\n- https://github.com/Leonidas-from-XIV/node-xml2js/issues/663\n- https://github.com/Leonidas-from-XIV/node-xml2js/pull/603/commits/581b19a62d88f8a3c068b5a45f4542c2d6a495a5\n- https://github.com/advisories/GHSA-776f-qx25-q3cc","created":"2023-04-05T21:30:24.000Z","reported_by":null,"title":"xml2js is vulnerable to prototype pollution ","npm_advisory_id":null,"overview":"xml2js versions before 0.5.0 allows an external attacker to edit or add new properties to an object. This is possible because the application does not properly validate incoming JSON keys, thus allowing the `__proto__` property to be edited.","url":"https://github.com/advisories/GHSA-776f-qx25-q3cc"}}}
```
Comment (vivek-sekhar) [inline yarn-audit-known-issues:1]: What is added in yarn audit?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts (hunk 1 above)
Comment (sealDebolina) [reply src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts]: @vivek-sekhar removed in latest build

File: src/main/steps/common/confirm-contact-details/checkanswers/ContactDetailsMapper.ts [hunk 3]
```diff
@@ -1,135 +1,238 @@
...
+//   if(partyDetails.isAtAddressLessThan5Years){
+//     req.session.userCase.isAtAddressLessThan5Years=partyDetails.isAtAddressLessThan5Years;
+//   }
+//   if (partyDetails.addressLivedLessThan5YearsDetails) {
+//     req.session.userCase.citizenUserAddressHistory = partyDetails.addressLivedLessThan5YearsDetails;
+//   }
 
-  return req.session.userCase;
-};
+//   return req.session.userCase;
```
[the remark this answers is not in the document]
Comment (sealDebolina) [reply src/main/steps/common/confirm-contact-details/checkanswers/ContactDetailsMapper.ts]: @vivek-sekhar removed in latest build

**Pull request merged**